feature/gribi/otg_tests/gribi_scaling/README.md

```diff
@@ -10,34 +10,31 @@ Validate gRIBI scaling requirements.
 *   Create 64 L3 sub-interfaces under DUT port-2 and corresponding 64 L3
     sub-interfaces on ATE port-2
 *   On DUT port-1 and ATE port-1 create a single L3 interface
-*   On DUT, create a policy-based forwarding rule to redirect all traffic received from DUT port-1 into VRF-1 (based on src. IP match criteria)
+*   On DUT, create a policy-based forwarding rule to redirect all traffic received from DUT port-1 into VRF-1 (based on src. IP match 111.111.111.111)
+*   On DUT, create a policy-based forwarding rule to redirect all traffic received from DUT port-1 into VRF-2 (based on src. IP match 222.222.222.222)
 *   Establish gRIBI client connection with DUT negotiating FIBACK as the
     requested ack_type and make it become leader.
-*   Using gRIBI Modify RPC install the following IPv4Entry sets, and validate
+*   TODO: Using gRIBI Modify RPC install the following IPv4Entry sets, and validate
```

Comment:
Is it better to have the code and README changes in the same PR, so that we don't need `TODO`? (and also avoid some confusion to readers while wait for the code change PR?)